src/Database/Query.php

```diff
@@ -1076,6 +1082,9 @@ public function order($fields, $overwrite = false)
      * This method allows you to set complex expressions
      * as order conditions unlike order()
      *
+     * Order fields are not suitable for use with user supplied data as they are
```

Comment:
plain text or text in note box(warning,..) for these notes?

Reply:
Is it possible to make a box here?

Reply:
I don't think so unless we start embedding HTML in docblocks.

Reply:
oh, I forgot  this repo is for `core` not `docs` ,i'm sorry :)